packages/manager/modules/telecom-dashboard/src/identity-check/form/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerTelecomDashboardIdentityCheckFormLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('identity-check.**', {
+      url: '/identity-check',
+      lazyLoad: ($transition$) => {
+        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+        return import(
+          /* webpackChunkName: "identity-check-form" */ './telecom-dashboard-identity-check-form.module'
```

Comment:
Using `webpackChunkName` is intended here? It seems to be effective also in production.

Reply:
Removed :+1: